plugins/outputs/cloudwatch/convert_otel.go

```diff
@@ -143,7 +143,7 @@ func ConvertOtelMetric(m pmetric.Metric) []*aggregationDatum {
 	name := m.Name()
 	unit, scale, err := cloudwatchutil.ToStandardUnit(m.Unit())
 	if err != nil {
-		log.Printf("W! cloudwatch: metricname %q has %v", name, err)
+		log.Printf("D! cloudwatch: metricname %q has %v", name, err)
```

Comment:
Since we silently convert JMX units to NONE, should we keep this as a warning message or it's still too noisy even before?

Reply:
You're right. There's no need to change it now.